features/step_definitions/stepdefs.js

```
@@ -17,6 +18,7 @@
 import jwt from 'jsonwebtoken';
 import Knex from 'knex';
 import jose from 'node-jose';
+import sharp from 'sharp';
```

Biome (1.9.4)

[error] This import is unused.

Unused imports might be the result of an incomplete refactoring.
Safe fix: Remove the unused import.


(lint/correctness/noUnusedImports)